sensors/bmp280/README.md

```diff
@@ -0,0 +1,23 @@
+
+# Connections
+
+| ESP32 | BMP280 |
+| ----- | ----------- |
+| GPIO22 | SCL |
+| GPIO23 | SDA |
+| 3.3V | Vin |
+| GND | GND |
+
+# Libraries Used
+
+- Adafruit BMP 280 Library
+
+(Refer [docs/to_be_added.md]() for help installing libraries)
```

Comment:
## Minor Changes

1. As the aim for the repository is to stand as a learning module / tutorial; it would be a better idea to expand or add a key to abbreviations to the aforementioned table. 
2. Line 15 can be commented out.